src/kde/test/tstquick_index.cc

```diff
@@ -0,0 +1,1731 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   kde/test/tstquick_index.cc
+ * \author Mathew Cleveland
+ * \date   Aug. 10th 2021
+ * \brief  quick_index testing function
+ * \note   Copyright (C) 2018-2021 Triad National Security, LLC.
```

Comment:
I know this is annoying. But please update copyright dates in all of your files to only list `2021`.

Reply:
Will do. Sorry I though I caught all of these but I guess I missed a few.